src/Components/cards/card.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useProduct } from "../../context/product-context"
+import { favourite_icon } from "../../Assets/index";
+import "./card.css";
+function ProductCard() {
+  const { items } = useProduct();
+ 
+
+  return (
+    <div>
+      {items.map((item) => (
+        <div className="product-grid">
+          <div class="card-exit">
+            <div class="card-img">
+              <img
+                src="https://assets.myntassets.com/h_720,q_90,w_540/v1/assets/images/16045848/2021/12/13/c42ef067-9fbe-4077-a48c-fca6331796ab1639383810735-Nike-Court-Vision-Low-Mens-Shoes-7251639383810387-1.jpg"
```

Comment:
Readability suggestion : Maybe consider having this image in your assets only rather than this long path or else consider using short url.